### ISSUE TITLE
Unable to login to alerta web UI

### ISSUE THREAD

**Issue by kavanat** [alerta/alerta#1503]
**Issue Summary**
When I load alerta we Ui and try to login I get this error
`Object of type bytes is not json serializable (500)`

**Environment**
- OS: [Linux]
- API version: [8.5.0]
- Deployment: [Self-hosted]

- Database: [Postgres]

- Server config:
  Auth enabled? Yes
  Auth provider? Basic
  Customer views? Yes
  (or provide the `/config` endpoint output)

- web UI version: [8.5.0]

Console screenshot

![1618944749641](https://user-images.githubusercontent.com/13285594/115449372-5e5db580-a238-11eb-8f15-edc6fbc12536.jpg)

**Comment by satterly** [comment]
Your deployment is self-hosted so you are going to have to provide all your configuration files and the commands you run to start the API and the web UI if you want me to help you.

**Comment by satterly** [comment]
This issue is blocked waiting on more information. The description or subsequent comments do not provide enough information to triage, investigate or resolve the issue. Please review the description against the [issue template](https://raw.githubusercontent.com/alerta/.github/master/.github/ISSUE_TEMPLATE/bug_report.md) and ensure all relevant information is included. If you do not know what is expected you can ask on [Slack](https://slack.alerta.dev).

**Comment by kavanat** [comment]
We have installed for our company and hence I'm unable to share all config files here

**Comment by satterly** [comment]
You need to help me to help you. Just redact any sensitive configuration values. 😕

**Comment by satterly** [comment]
Seeing that you have opened a new issue I assume this one is resolved #1504
